src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1438,7 +1446,7 @@ public function testRawCommandLine()
 (
     [0] => -
     [1] => a
-    [2] => 
```

Comment:
Needs to be reverted. I've stepped into that trap multiple times myself.

Reply:
Oops, this is strange.. It is automatically "fixed" by .editorconfig. I did a permanent fix.